visualSHARK/base.py

```diff
@@ -0,0 +1,5 @@
+REST_FRAMEWORK = {
```

Comment:
There is already a REST_FRAMEWORK config in settings/base.py that is the one we should modify.
We only need to add the new CustomPermission there.